include/cantera/thermo/WaterSSTP.h

```diff
@@ -238,6 +240,9 @@ class WaterSSTP : public SingleSpeciesTP
         return m_waterProps.get();
     }
 
+    //! Switch that enables calculations in the gas phase
+    void allowGasPhase(bool flag) { m_allowGasPhase = flag; }
```

Comment:
I'm worried about the addition of this member function, because it implies that the `m_allowGasPhase` flag actually works, which I think isn't true. As an example, if this flag is set to `true`, setting T and P to (300, 101325) and then (400, 101325) leads to a density of 937.411 kg/m^3, which is clearly wrong.

Reply:
I really didn’t want to implement this either, but an example in the test suite depends on it. I’d certainly refuse to add it to Python ...

Reply:
Beyond adding a longer docstring, I don't have a good alternative idea, as I really do not want to break the test suite.

Reply:
Taking a second look at this, I think I'd settle for naming this function `_allowGasPhase`, using the leading underscore to indicate that this function is in some sense internal and not to be used.

Reply:
done!